### ISSUE TITLE
suggested update on the Installation guide for Scoop@Windows

### ISSUE THREAD

**Issue by remifan** [FastNFT/FNFT#72]
in the the 3-step installation via `scoop`
```
scoop install git
scoop install cmake
scoop install gcc
```
`scoop install gcc` can install `gcc` now but is neither sufficient nor safe.

Due to the msys2's changes in compression method, Scoop had long been [failed to install `gcc` (msys2's mingw build by default)](https://github.com/ScoopInstaller/Main/issues/1752#issuecomment-761114004). As a breaking change, the recent version of Scoop has migrated to the [WinLib's build of gcc](https://github.com/ScoopInstaller/Main/pull/2967) for easier maintenance.

To stay with mingw64 toolchain without leaving CLI, here are working steps without Scoop's bucket hack:

launch  PowerShell/CMD, then
```
# same
scoop install git
scoop install cmake

# change gcc to mingw
scoop install mingw
```

now everything should be ready to build FNFT. If Matlab is installed, you should see `Matlab is found` log during `cmake`, and `*.mex` files should be present in the `build` folder after successful building.

**Comment by wahls** [comment]
Thanks for the comment. These instruction indeed need updating with the next release. Unfortunately we do not have a lot of Windows users around here at the moment. 

One question: Does this also work with Matlab. That is, does it build usable mex files?

**Comment by remifan** [comment]
> Thanks for the comment. These instruction indeed need updating with the next release. Unfortunately we do not have a lot of Windows users around here at the moment.
> 
> One question: Does this also work with Matlab. That is, does it build usable mex files?

Yes, it is working with Matlab, I just checked all the example `.m` files which are running as expected. Also all tests are passed after building.

I have tested this installation method on 2 Windows 11 PCs, it should be working for Windows 10 too.

**Comment by xuchuang7916** [comment]
> > Thanks for the comment. These instruction indeed need updating with the next release. Unfortunately we do not have a lot of Windows users around here at the moment.
> > One question: Does this also work with Matlab. That is, does it build usable mex files?
> 
> Yes, it is working with Matlab, I just checked all the example `.m` files which are running as expected. Also all tests are passed after building.
> 
> I have tested this installation method on 2 Windows 11 PCs, it should be working for Windows 10 too.

@remifan Thanks a lot for your comment, I followed your suggestion and installed FNFT on Windows 10 successfully just now.

**Comment by wahls** [comment]
Dear @xuchuang7916, thank you for reporting.

**Comment by wahls** [comment]
Resolved in version 0.5.0.

**Comment by wahls** [comment]
Resolved in version 0.5.0.
